### ISSUE TITLE
Feature Request: support for image embeddings

### ISSUE THREAD

**Issue by andrefrcastro** [theodo-group/LLPhant#27]
This would enable similarity searches for images. It requires generating vector representations for images and storing them efficiently. This would be great for image-based searches.

**Comment by MaximeThoonsen** [comment]
@andrefrcastro I agree it would be cool. What do you use to generate the embeddings of images for now?

**Comment by andrefrcastro** [comment]
@MaximeThoonsen [OpenAI](https://platform.openai.com/docs/guides/embeddings)
